modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -2078,8 +2078,8 @@ GAPI_FLUID_KERNEL(GFluidResize, cv::gapi::imgproc::GResize, true)
        int outSz_h;
        if (outSz.width == 0 || outSz.height == 0)
        {
-           outSz_w = static_cast<int>(round(in.size.width * fx));
-           outSz_h = static_cast<int>(round(in.size.height * fy));
+           outSz_w = saturate_cast<int>(in.size.width * fx);
+           outSz_h = saturate_cast<int>(in.size.height * fy);
```

Comment:
Does this change anything? Do tests pass now?

Reply:
That change put the rounding of output size in accordance with it in G_TYPED_KERNEL GResize. It has no affect on tests